### PR TITLE
187434183 set up flow to create course

### PR DESCRIPTION
**Pull request by cynthia-lixinyi**
# General Info

- [Pivotal Tracker Story](https://www.pivotaltracker.com/story/show/########)
- [ ] Breaking?

# Changes

Explain your changes here (in such a way that you would understand why you made them a year from now).
Implemented `create` method in `class CoursesController`. This method intends to create an entry in the courses table.

# Testing

Explain how you tested your changes.  If testing is not necessary, explain why.
Two cases are tested:
(1) A course with the provided course_name is created successfully.
(2) A course with the same course name already exists. In this case, an error message saying "A course with the same course name already exists." shows up in the JSON response body

# Documentation

Does this PR require documentation.  If so, explain where it can be found.
N/A

# Checklist

- [x] Name of branch corresponds to story

### CODE REVIEW

File: app/controllers/api/v1/courses_controller.rb [hunk 1]
```diff
@@ -1,8 +1,25 @@
 module Api
   module V1
     class CoursesController < BaseController
+      require 'lms_api'
```
Comment (Connor-Bernard) [inline app/controllers/api/v1/courses_controller.rb:4]: What do we use this for?  I thought we were going to try to use the canvas facade

Comment (Connor-Bernard) [reply app/controllers/api/v1/courses_controller.rb:4]: Also, how are we making the call to canvas?

File: app/views/courses/index.html.erb
Comment (Connor-Bernard) [inline app/views/courses/index.html.erb:1]: Do we want to do any UI right now for the API?

**Review by Connor-Bernard — CHANGES_REQUESTED** [review]
Made a couple of comments

**Review by cynthia-lixinyi — COMMENTED** [review]
~~Looks good to me.~~ Sorry, my intention was to review another PR.

**Review by Connor-Bernard — APPROVED** [review]
LGTM

**Pull request merged**